.github/workflows/release_otelbench.yaml

```diff
@@ -0,0 +1,106 @@
+run-name: release-otelbench
+name: Release otelbench docker image on elastic container library
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version of the otelbench image to release, should follow format "vN.N.N".
+        required: true
+  push:
+    branches:
+      - main
+    paths:
+      - ./loadgen/cmd/otelbench/Makefile
+
+jobs:
+  check-release:
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get-new-version.outputs.new_version }}
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+
+      - name: Checks if release is required.
+        id: get-new-version
+        run: ./scripts/check_release.sh "${{ github.event_name }}" "${{ github.event.inputs.version }}"
+
+  release-otelbench:
+    timeout-minutes: 30
+    runs-on: ubuntu-latest
+    needs:
+      - check-release
+    if: needs.check-release.result == 'success'
+    permissions:
+      contents: write
+    env:
+      ELASTIC_REGISTRY: "docker.elastic.co/observability-ci"
+      IMAGE_NAME: "otelbench"
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: '0'
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version-file: go.mod
+          cache: 'false'
+
+      - name: Cache go tools
+        id: go-tools-cache
+        uses: actions/cache@v4
+        with:
+          path: ./.tools
+          key: go-tools-${{ runner.os }}-${{ hashFiles('internal/tools/go.sum') }}
+
+      # If there is no cache with the go tools,
+      # install them. We need the changelog
+      # tool for this workflow to work.
+      - name: Install tools
+        if: steps.go-tools-cache.outputs.cache-hit != 'true'
+        run: |
+          make install-tools
+
+      - name: Update otelbench changelog and delete changelog fragments
+        working-directory: ./loadgen/cmd/otelbench
+        run: ../../../.tools/chloggen update --version ${{ needs.check-release.outputs.version }}
```

Comment:
Do you think `/.tools` could be added to the PATH so the relative path can be simplified?

```yaml
      - name: Add .tools to PATH
        run: echo "$GITHUB_WORKSPACE/.tools" >> $GITHUB_PATH

      - name: Update otelbench changelog and delete changelog fragments
        working-directory: ./loadgen/cmd/otelbench
        run: ./chloggen update --version ${{ needs.check-release.outputs.version }}
```

Reply:
Thanks, I didn't know this existed. Fixed it.